faunadb-java/src/test/java/com/faunadb/client/ClientSpec.java

```diff
@@ -1180,19 +1180,6 @@ public void shouldEvalFormatExpression() throws Exception {
     );
   }
 
-  @Test
-  public void shouldEvalContainsExpression() throws Exception {
-    Value contains = query(
-      Contains(
```

Comment:
@nathaniel-may I think we agreed to returns the `Contains` function and remove it later, right?

Reply:
This function is not available in API V4, I had to remove the test to get a green result :traffic_light: